src/Memoize/Drivers/MemoryDriver.php

```diff
@@ -6,28 +6,29 @@
 
 use Closure;
 use StellarWP\Arrays\Arr;
+use StellarWP\Memoize\Contracts\DriverInterface;
 
-class MemoryDriver implements Contracts\DriverInterface
+class MemoryDriver implements DriverInterface
```

Comment:
I was just copying the code before, but I think we should make this final and of course the `$cache` property private.

Developers could then make a decorator if they want to modify the functionality of this, using composition instead of inheritance.

I could provide an example if you want one for the docs...